demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -891,3 +897,138 @@ def test_IsContextPathChangedValidator_remove_command():
         f"Command {command_name} has been removed from the integration. This is a breaking change, and is not allowed."
         in errors[0].message
     )
+
+
+def setup_integration_with_3_commands_and_5_args():
```

Comment:
Let's remove this function, I don't think that a helper function to init the args is a good idea.
Also, it's done wrong here, there's a better way to init an integration with 3 commands.
see 'test_IsContainingMultipleDefaultArgsValidator_is_valid' in IN_Validators as a reference.

Reply:
You rather it as a global? and not a function?